content/docs/components/serving/tfserving_new.md

```diff
@@ -6,32 +6,150 @@ weight = 51
 
 ## Serving a model
 
-We treat each deployed model as two [components](https://ksonnet.io/docs/tutorial#2-generate-and-deploy-an-app-component)
-in your APP: one tf-serving-deployment, and one tf-serving-service.
-We can think of the service as a model, and the deployment as the version of the model.
+Below is an example of a yaml manifest that deploys a mnist model for serving.
 
-Generate the service(model) component
-
-```
-ks generate tf-serving-service mnist-service
-ks param set mnist-service modelName mnist    // match your deployment mode name
-ks param set mnist-service trafficRule v1:100    // optional, it's the default value
-ks param set mnist-service serviceType LoadBalancer    // optional, change type to LoadBalancer to expose external IP
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: mnist
+    ksonnet.io/component: mnist-service
+  name: mnist-service
+  namespace: kubeflow
+spec:
+  ports:
+  - name: grpc-tf-serving
+    port: 9000
+    targetPort: 9000
+  - name: http-tf-serving
+    port: 8500
+    targetPort: 8500
+  selector:
+    app: mnist
+  type: ClusterIP
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: mnist
+    ksonnet.io/component: mnist-v1
+  name: mnist-v1
+  namespace: kubeflow
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true"
+      labels:
+        app: mnist
+        version: v1
+    spec:
+      containers:
+      - args:
+        - --port=9000
+        - --rest_api_port=8500
+        - --model_name=mnist
+        - --model_base_path=gs://kubeflow-examples-data/mnist
```

Comment:
I might suggest using a local path like "/some/path/to/model"

You can then have separate sections below talking about loading models from GCS, S3, etc...